tests/topotests/bgp-ecmp-topo2/test_ebgp_ecmp_topo2.py

```diff
@@ -63,7 +63,7 @@
     reset_config_on_routers,
 )
 from lib.topolog import logger
-from lib.bgp import verify_bgp_convergence, create_router_bgp, clear_bgp_and_verify
```

Comment:
So does this mean that clear_bgp_and_verify() is ... unreliable? it is still being used in other test suites.

Reply:
Hi Mark, This API was not designed not to be used with @retry, but I have put the support bundles/data dumps logic in such a way that when a verification fails after all possible retried, it generates support bundles. 

So it not unreliable, the only thing is that it will not allow to generate support bundles as per current design. I will take care of replacing this API in other tests as well, where ever it is being used.

Reply:
@mjstapp , Moreover I wanted to generate support bundles for this particular test failure so I changed the API. All old test cases are working fine with clear_bgp_and_verify() API. I think we can leave old tests as it is. Please suggest.